public/app/features/sandbox/TestStuffPage.tsx

```diff
@@ -150,4 +154,23 @@ export function getDefaultState(): State {
   };
 }
 
+function LinkToBasicApp({ linkId }: { linkId: string }) {
+  const { link } = getPluginLink({
+    id: linkId,
+    queryParams: {
+      title: 'something',
+    },
+  });
```

Comment:
I really dont understand this getPluginLink function and how it's intended to be used. I am missing a "link type" enum that let's Grafan know where to use the link

Reply:
It will primary be used by plugins/alerting to link to other plugins. The possibility to extend parts of Grafana with links or commands will be added in follow up PRs.